config/dynflow/worker.yml.example

```diff
@@ -0,0 +1,4 @@
+:concurrency: 10
+:queues:
+  - default
+  - remote_execution
```

Comment:
Given this can vary per worker, is there anywhere that allows monitoring of how many workers there are running, what their concurrency is and what queues they are each monitoring other than looking at the configuration files? I'm wondering if the Ping API reports this information, or the About page includes this information for admins or the dynflow console allows monitoring this.

Reply:
Sidekiq provides its own monitoring console which will eventually be mounted on `/foreman_tasks/sidekiq` (the same way as dynflow console is) and will be accessible only by admin, All the information should be displayed there.

Reply:
This mounts the console: theforeman/foreman-tasks#468